distros/kubernetes/nvsentinel/charts/kubernetes-object-monitor/templates/clusterrole.yaml

```diff
@@ -0,0 +1,31 @@
+# Copyright (c) 2025, NVIDIA CORPORATION.  All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: {{ include "kubernetes-object-monitor.fullname" . }}
+  labels:
+    {{- include "kubernetes-object-monitor.labels" . | nindent 4 }}
+rules:
```

Comment:
If the user is monitoring custom resources, don't we also have to add read only RBAC for those too? Can we take as input the resources in the helm values (if not done already) and add logic in the helm chart to add the required read only rbac?

Reply:
rbac will be auto-generated now. Added an example of how in PR description